guidelines/_sources/04_report-prompts.tex

```diff
@@ -17,13 +20,20 @@ \subsubsection{Recommendations}
+When reporting prompts, researchers \textbf{MUST} also reference the model version as specified in Section X ('Report Model Version and Configuration'), as prompt effectiveness varies across model versions. For example: \textit{This prompt performed differently with GPT-4 (effective) versus Llama 2 (less effective) despite identical parameters.}
+
+Prompt development is often iterative, involving collaboration between human researchers and AI tools. Researchers \textbf{SHOULD} report any instances where LLMs were used to suggest prompt refinements, as well as how those suggestions were incorporated. Furthermore, prompts may need to be revised in response to failure cases where the model produced incorrect or incomplete outputs. Iterative changes based on human feedback and pilot testing results should also be included in the documentation. A prompt changelog can help track and report the evolution of prompts throughout a research project, including key revisions, reasons for changes, and versioning (e.g., v1.0: initial prompt; v1.2: added output formatting; v2.0: incorporated examples of ideal responses).
+\comment{We might want to add some sort of guidance on tracking and reporting prompt evolution throughout research projects? Such as recommendation for maintaining a \textit{prompt changelog}. For example: - Initial design (v1.0): initial prompt text; - Key revision (v1.2): Added specific output formatting requirements; - Final version (v2.0): Incorporated examples of ideal responses.}
 
 Finally, pilot testing and prompt evaluation are vital for ensuring that prompts yield reliable results. If such testing was conducted, researchers \textbf{SHOULD} summarize key insights, including how different prompt variations affected output quality and which criteria were used to finalize the prompt design.
 
+\comment{Maybe the document could establish a connections to guidelines in  `Report Model Version and Configuration'? Something along this line? ``When reporting prompts, researchers \textbf{MUST} also reference the model version as specified in Section X ('Report Model Version and Configuration'), as prompt effectiveness varies across model versions. For example: \textit{This prompt performed differently with GPT-4 (effective) versus Llama 2 (less effective) despite identical parameters.}''.}
+
 \subsubsection{Example(s)}
 A debugging study may use a prompt structured like this:
 
+\comment{Do you think a larger variety of examples would be beneficial? Something from requirements engineering, code generation, and testing?}
```

Comment:
Yes, but I'd also like to keep the example small enough so that *all* of it can fit into the guidelines to avoid any doubts about what aspect of a prompt can be omitted if there's lack of space. So I'm keeping this one for now.